src/path.h

```diff
@@ -24,6 +24,7 @@ constexpr bool IsWindowsDeviceRoot(const char c) noexcept;
 #endif  // _WIN32
 
 void ToNamespacedPath(Environment* env, BufferValue* path);
+void FromNamespacedPath(std::string* path);
```

Comment:
Nit: This signature is a bit awkward. Could this potentially be modified to accept a `std::string_view` and return a `std::string_view`?

Reply:
Yes, but it is a noop for non-windows. It will not be a noop after this nit.